### ISSUE TITLE
[App Support]

### ISSUE THREAD

**Issue by Khoriggan** [beardedspice/beardedspice#161]
Hi there, 

I'm completely new here, tried to understand how I can implement and add a site but I really didn't get anything. I've never programmed before that's probably why. 
I was wondering if it's possible to configure Bearded Spice to make it work with this website.
http://www.letournedisque.com

Many thanks in advance

**Comment by frdmn** [comment]
I'll see what I can do :)

**Comment by frdmn** [comment]
Got it working. Now we need to wait for @Stillness-2 or someone else with merge access. 

You can build the fork here if you are impatient :P

https://github.com/frdmn/beardedspice

**Comment by foozee** [comment]
thanks @frdmn

**Comment by frdmn** [comment]
@foozee Added it. I assume it should return true if the tab is currently playing a track and false if it isn't, right? I didn't figure out what it really does yet. :P

**Comment by foozee** [comment]
yes, it does exactly that, this function will help the *auto* mode to figure out *smartly* which tab is currently playing and direct the media action (play/pause/next/prev) at it (in case of no tab being explicitly selected by the user) :)

**Comment by foozee** [comment]
thanks a lot @frdmn

**Comment by Khoriggan** [comment]
Many thanks, I’m really impressed how fast it was.
